codejail/safe_exec.py

```diff
@@ -22,7 +22,7 @@
 # Flags to let developers temporarily change some behavior in this file.
 
 # Set this to True to log all the code and globals being executed.
-LOG_ALL_CODE = False
+LOG_ALL_CODE = True
```

Comment:
I presume this change should be reverted before merging?

Reply:
Yes.